android/fastlane/Fastfile

```diff
@@ -0,0 +1,180 @@
+# This file contains the fastlane.tools configuration
+# You can find the documentation at https://docs.fastlane.tools
+#
+# For a list of all available actions, check out
+#
+#     https://docs.fastlane.tools/actions
+#
+# For a list of all available plugins, check out
+#
+#     https://docs.fastlane.tools/plugins/available-plugins
+#
+
+# Uncomment the line if you want fastlane to automatically update itself
+# update_fastlane
+
+default_platform(:ios)
+
+skip_docs
+
+# CONFIG VARIABLES
+app_name = 'ReactNativeBase'          
+folder = ENV["FOLDER"]
+s3_bucket = ENV["BUILD_BUCKET"]
+s3_region = ENV["AWS_REGION"]
+s3_key = ENV["AWS_ACCESS_KEY_ID"]
+s3_secret_key = ENV["AWS_SECRET_ACCESS_KEY"]
+slack_url = ENV["SLACK_URL"]
+slack_channel = ENV["SLACK_CHANNEL"]
+json_key = ENV["JSON_KEYFILE"]
+
+default_platform(:android)
+
+lane :build do |options|
+  # increment version
+  version = get_version_name(
+      gradle_file_path: './app/build.gradle'
+  ).to_s
+  build_number = increment_version_code(
+      gradle_file_path: './app/build.gradle'
+  ).to_s
```

Comment:
if I release a version to QA and then subsequently release the same version to staging, will they have different build_numbers? Is that the intended behaviour?

Reply:
from what I've seen so far build numbering across targets do not keep much relation to each other. But in any case I am thinking about removing the increment_version_code altogether and leave this for manual update as we are not commiting this (won't play well with CICD) and it doesn't really add value otherwise